app/helpers/api/admins_helper.rb

```diff
@@ -0,0 +1,2 @@
+module Api::AdminsHelper
```

Comment:
Could you explain what this module will be used for?

Reply:
This is obsolete.  I was messing around before following a tutorial and we decided to go with User as the model.

Reply:
Is it as simple as git rm?